web/src/components/Popup/MiniGuides/Staking/Notifications.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import styled from "styled-components";
 
-import NotificationsSvg from "tsx:assets/svgs/mini-guides/staking/notifications.svg";
+import NotificationsSvg from "assets/svgs/mini-guides/staking/notifications.svg";
```

Comment:
tiny thing (and optional), but these can be made "svgs/mini-guides...." directly. without the "assets"